sanic/sanic.py

```diff
@@ -176,6 +177,15 @@ def converted_response_type(self, response):
         :return: Nothing
         """
         try:
+            request_started.send(request)
```

Comment:
Does it perform this on every request, even if there are no signal handlers for `request_started`? How does that impact performance? Results from running `wrk` before and after on the simple_server.py would be good to see.